src/services/coreBackService.js

```diff
@@ -28,10 +28,10 @@ function generateAxiosConfig(req) {
   };
 }
 
-function postJourneyEvent(req, event) {
+function postJourneyEvent(req, event, queryParams = {}) {
   return axiosInstance.post(
     `${API_JOURNEY_EVENT}/${event}`,
-    {},
+    queryParams,
```

Comment:
Have you tested this? I think the second argument here is the request body, not query params.

You might need to do something like:
```
function postJourneyEvent(req, event, currentPage) {
  return axiosInstance.post(
    `${API_JOURNEY_EVENT}/${event}`,
    {},
    {
      ...generateAxiosConfig(req),
      params: currentPage ? { currentPage } : {},
  );
```

Reply:
Ahh the build failed 🤦  thanks I'll change now